Cargo.lock

```diff
@@ -1173,6 +1174,23 @@ dependencies = [
  "tlvc",
 ]
 
+[[package]]
+name = "humility-cmd-console-proxy"
+version = "0.1.0"
+dependencies = [
+ "anyhow",
+ "clap 3.2.21",
```

Comment:
Unrelated to this PR, but seeing a version number made me realize that we have an unused `structopt` dependency that's dragging in a _different_ `clap` version.  I opened #292 to fix that.